blocks/rest/src/edit.js

```diff
@@ -95,6 +95,12 @@ export default function Edit( { attributes, setAttributes, clientId } ) {
 		});
 	}
 
+	function decodeHtml(text) {
```

Comment:
I thought surely there must be an easier way, but after looking it up I see why this is popular. Do either of the options below strike you as less code in any useful way? If you want to stick with the current version, could you add a link or some function documentation about why we're doing this?

Since lodash comes with WordPress, we could also use `_.unescape` 
https://lodash.com/docs/4.17.15#unescape

or if we're really only concerned with `&amp;` it could be solved with `.replaceAll('&amp;', '&')`

Reply:
Thanks! I decided to switch to `_.unescape` which still seems to work fine. I just pushed to develop again too.

Reply:
One more thing about this, I feel lodash should be listed in the dependencies. 

If you add `import { unescape } from 'lodash'` at the top, does `wp-scripts` add it to the dependencies array?

Reply:
It does @douglas-johnson, just added that to `edit.js`